pkg/collector/python/datadog_agent.go

```diff
@@ -287,14 +294,109 @@ var defaultSQLPlanNormalizeSettings = traceconfig.JSONObfuscationConfig{
 		"using_join_buffer",
 		"using_temporary_table",
 		// postgres
+		"Actual Loops",
+		"Actual Rows",
+		"Actual Startup Time",
+		"Actual Total Time",
 		"Alias",
+		"Async Capable",
+		"Average Sort Space Used",
+		"Cache Evictions",
+		"Cache Hits",
+		"Cache Misses",
+		"Cache Overflows",
+		"Command",
+		"Conflicting Tuples",
+		"Constraint Name",
+		"CTE Name",
+		"Custom Plan Provider",
+		"Deforming",
+		"Emission",
+		"Exact Heap Blocks",
+		"Execution Time",
+		"Expressions",
+		"Foreign Delete",
+		"Foreign Insert",
+		"Foreign Update",
+		"Function Call",
+		"Function Name",
+		"Generation",
+		"Group Count",
+		"Grouping Sets",
```

Comment:
I'm not entirely sure but should this one pass through the obfuscator?

https://github.com/postgres/postgres/blob/master/src/backend/commands/explain.c#L2400-L2428

Reply:
Discussed offline, but for future readers. `Grouping Sets` looks to only reveal column names, so obfuscation is unnecessary.
Example:
```
SELECT
    c1,
    c2,
    aggregate_function(c3)
FROM
    table_name
GROUP BY
    GROUPING SETS (
        (c1, c2),
        (c1),
        (c2),
        ()
);
```